### DISCUSSION TITLE
How to apply bootstrap's javascript and css at stackblitz

### DISCUSSION THREAD

**Discussion by jt4000** [twbs/bootstrap#34661] [Q&A]
Goal:
Apply bootstrap 5:s javascript and css at syntax import at stackblitz

Problem:
How should I write the bootstrap 5's css and java at import syntax at stackblitz?

Stackblitz:
https://stackblitz.com/edit/react-u6qgdk?

Info:
New in ReactJS

Thank you!

Code:
```

import React from 'react';
import './style.css';

export default function App() {
  return (
    <div class="container">
      <div class="row">
        <div class="col">
          Column
        </div>
        <div class="col">
          Column
        </div>
        <div class="col">
          Column
        </div>
      </div>
    </div>
  );
}
```

**Comment by Hiws** [comment]
For the CSS you can just import it using
`import 'bootstrap/dist/css/bootstrap.min.css'`
The same way you're importing your `style.css`
 
For the JavaScript you can take a look at 
https://getbootstrap.com/docs/5.1/getting-started/parcel/#importing-javascript

Though I'm unsure how well that will work together with React.
